src/components/Progress/ASABProgress.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+
+import './ASABProgress.scss';
+
+export function ASABProgress({
+	showLabel = true,
+	children,
+	...props
+}) {
+	// Value has to be present and numeric
+	let progressValue = props?.value;
+
+	// Early return if not a number
+	if (!(typeof progressValue === 'number')) {
+		return null;
```

Comment:
we should at least write this to console, that the progress is not rendered due to a invalid value